xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,304 @@ migrate</screen>
     the selected nodes.
    </para>
   </sect2>
+  <sect2 xml:id="sec.depl.maintenance.upgrade.troubleshooting">
+   <title>Common Upgrade Problems and Possible Solutions</title>
+    <qandaset>
+     <qandaentry>
+      <question>
+       <para>
+	Upgrade of the admin server has failed.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Check for empty, broken, and not signed repositories in the &admserv;
+	upgrade log file <filename>/var/log/crowbar/admin-server-upgrade.log</filename>. Fix the
+	repository setup. Upgrade then remaining packages manually to &cloudos;
```

Comment:
What does "fix the repository setup" mean?

Reply:
We don't really know what went wrong, that's why the advice is generic. It's possible repositories are empty for example.
User should know what "fix the repo setup" means as he/she was supposed to prepare those repos before

Reply:
Hmm should we make this clear? Or do we assume it'll be clear enough?